carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/log/ThreadLogAppender.java

```diff
@@ -100,4 +108,12 @@ public void close() {
     public boolean requiresLayout() {
         return false;
     }
+
+    private void ensureCapacity(int len) throws IOException {
+        long newBytesWritten = this.bytesWritten + len;
+        long maxBytes = this.maxBytes *1024 * 1024;
+        if (newBytesWritten > maxBytes)
+            throw new IOException("File size exceeded: " + newBytesWritten + " > " + this.maxBytes);
```

Comment:
let's update message:
"test Log file size exceeded core limit: " + newBytesWritten + " > " + this.maxBytes